embedded-graphics/src/draw_target/mod.rs

```diff
@@ -586,4 +651,11 @@ where
     fn clipped(&mut self, area: &Rectangle) -> Clipped<'_, Self> {
         Clipped::new(self, area)
     }
+
+    fn convert_color<C>(&mut self) -> ConvertColor<'_, Self, C>
```

Comment:
This name sounds a bit strange, especially in the context of `thing.draw(&mut display.convert_color())`. The best alternative I can come up with so far is `color_converted` which would match the `-ed` suffixes of the other display modes. If we change this method name, the trait name should probably be changed to `ColorConverted` too.

Open to better suggestions - `color_converted` isn't great.

Reply:
I don't have any better ideas, so I've used the names you've suggested.